src/custom/tokens/rinkeby-token-list.json

```diff
@@ -0,0 +1,100 @@
+{
+    "name": "GP Rinkeby",
+    "version": {
+      "major": 0,
+      "minor": 1,
+      "patch": 0
+    },
+    "keywords": [
+      "GP",
+      "Rinkeby",
+      "tokens",
+      "trusted"
+    ],
+    "logoURI": "https://avatars.githubusercontent.com/u/24954468?s=200&v=4",
+    "timestamp": "2021-02-22T13:09:54.363Z",
+    "tokens": [
+      {
+        "name": "Gnosis",
+        "chainId": 4,
+        "symbol": "GNO",
+        "decimals": 18,
+        "address": "0xd0Dab4E640D95E9E8A47545598c33e31bDb53C7c",
+        "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6810e776880C02933D47DB1b9fc05908e5386b96/logo.png"
+      },
+      {
+        "name": "USDT",
+        "chainId": 4,
+        "symbol": "USDT",
+        "decimals": 18,
```

Comment:
can u review the decimals? some of them are wrong

Reply:
oh true, good catch